src/main/grammars/ElmParser.bnf

```diff
@@ -224,15 +224,15 @@ TypeRef ::= UpperCaseQID (SingleTypeExpression)+
 
 private SingleTypeExpression ::=
     TypeRefWithoutArgs
-    | TypeVariableRef
+    | TypeVariable
```

Comment:
Nice. The `Ref` suffix is leftover from when I was trying to make a distinction between where types are used vs declared. It was a bad idea since it suggests a relationship to `PsiReference`, so thank you for fixing it here.